lib/galaxy/tools/parameters/basic.py

```diff
@@ -2340,8 +2278,8 @@ def _get_select_dataset_collection_field( self, trans, history, multiple=False,
             name = history_dataset_collection.name
             hid = str( history_dataset_collection.hid )
             hidden_text = ""  # TODO
-            subcollection_type = self._history_query( trans ).can_map_over( history_dataset_collection ).collection_type
-            id = "%s|%s" % ( dataset_matcher.trans.security.encode_id( history_dataset_collection.id ), subcollection_type )
+            subcollection_type = self._history_query( trans ).collection_type_description.collection_type
```

Comment:
Bad merge here - definitely want the newer version of this - ``can_map_over( history_dataset_collection ).collection_type``